app/src/main/kotlin/com/wire/android/ui/home/conversations/usecase/GetMessagesForConversationUseCase.kt

```diff
@@ -50,11 +50,12 @@ class GetMessagesForConversationUseCase @Inject constructor(
         val pagingConfig = PagingConfig(
             pageSize = PAGE_SIZE,
             prefetchDistance = PREFETCH_DISTANCE,
-            initialLoadSize = max(INITIAL_LOAD_SIZE, lastReadIndex + PREFETCH_DISTANCE)
+            initialLoadSize = INITIAL_LOAD_SIZE
```

Comment:
We are making sure that the first load size will always be `50` as per the value on the variable